item-spec/item-spec.md

```diff
@@ -110,7 +108,7 @@ link is required to be absolute.
 
 #### Datasets
 
-Items are *strongly recommended* to provide a link to a dataset definition.
+Items are *required* to provide a link to a dataset definition.
```

Comment:
Maybe it's a bad idea, but I was thinking we could bring the *requirement* to link to a dataset in at the Catalog level (or the Dataset level). Like let the item spec stand alone, and just have a strong recommendation. And then for Catalog it could add that any Item referenced from the Catalog should have a Dataset.

As I've been explaining the set of specs I've been thinking about trying to make sure they could be used standalone. And requiring a dataset link in an Item does make it less able to stand on its own. 

And I think we should put a bit more color here on why this is so strongly recommended. Explain that it provides the license and provider information, as well as additional information about the set of Items.

Reply:
I think that idea doesn't work out as it doesn't really care where things are written down as required - they are still required then. So if we don't want close coupling of things, we can just say that they are strongly recommended and must avoid using requirements between the specs at all. Sure, we can add that items that are linked from in a dataset are required to link back to its dataset, but that's a different thing for me than what you wrote in your recap: "This also lead to requiring a dataset link in every Item". So let's make it that items linked from a dataset requre a link back. I'll work on that next week.

Reply:
Changed, please review it again.